samples/dotnet/clientcreds/Lenus.Samples.ClientCredentialsFlow/src/appsettings.Development.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "Lenus": {
```

Comment:
@ash-leigh - rather than having these config values blank can we have sandbox values defaulted to assist users in understanding what is expected.

See https://github.com/lenushealth/integration-samples/blob/main/samples/dotnet/clinorg/src/appsettings.Development.json for example

Reply:
I've added these now @Philo